ucfcs/ucf-ai-advising-chatbot
# NLU Entities Function

Completed a function called `get_entities()` which takes an input string, preprocesses it, and returns the top-three predicted entities.

As part of this pull request, the original terminal chatbot code was moved to a new `chatbot-testing.py` file, and `chatbot.py` now contains the new function.

**Testing the `get_entities()` Function**
1. Head to the AI directory: `cd backend/src/ai`
2. Open the Python terminal environment: `python`
3. Import the function: `from chatbot import get_entities`
4. Call the function and print the results: `print(get_entities(<INPUT>))`
5. The program should print a `1 x 3` array with the top-three entities predicted.

***Note:** This is designed to run with the first model. Please test this on the functionality of the function, rather than the accuracy of the model.*
